src/it/scala/org/ergoplatform/it/ThreeNodesSuite.scala

```diff
@@ -0,0 +1,36 @@
+package org.ergoplatform.it
+
+import com.typesafe.config.Config
+import org.scalatest.Suite
+
+import scala.collection.immutable.IndexedSeq
+
+class ThreeNodesSuite extends IntegrationSuite {
+
+  private val nodesCount = 3
+
+  def knownPeers(nodes: Map[String, Node], nodeConfig: Config) = {
+    val node01 = nodes.map(_._2).find(_.settings.scorexSettings.network.nodeName == "node01")
+    val node02 = nodes.map(_._2).find(_.settings.scorexSettings.network.nodeName == "node02")
+
+    val ls = List(node01, node02).flatten.map(_.settings.scorexSettings.network.nodeName)
+
+    if (nodeConfig.getString("scorex.network.nodeName") == "node02" && node01.isDefined) {
+      val n = node01.get
+      s" -Dscorex.network.knownPeers.0=${n.nodeInfo.networkIpAddress}:${n.nodeInfo.containerNetworkPort}"
```

Comment:
 it worth introducing helper method in `Docker` class for the known peers setting to avoid ugly code repitition